docs( config module )

tlerias opened the pull request:
Clarify documentation for passing config from one module to another.

akanass committed:
Merge pull request #20 from hapinessjs/next

Merge next to master

akanass committed:
Merge pull request #27 from hapinessjs/next

v1.0.0-beta.3

Commit:
docs( config module )

fauvillej commented:
Right, then we should remove the constructor of ExternalModule or use @Inject() as well.

    constructor(@Optional() config: Config) {
       ...
    }

In the service, @Optional() is required, because if you don't provide the config, the DI will throw an error if you don't use it.

Commit:
docs( clean up config docs )

tlerias commented:
@Juneil I updated the PR with a fix for your comment.
I also added a comment in the README code next to the @Optional, please review that text to make sure it is correct.

Commit:
docs( update docs further )

tlerias commented:
@Juneil fixed :)

fauvillej committed:
docs(API): typo fixes
